src/dataregistry/registrar/base_table_class.py

```diff
@@ -56,6 +56,7 @@ def __init__(self, db_connection, root_dir, owner, owner_type):
         # Database engine and dialect.
         self._engine = db_connection.engine
         self._schema = db_connection.schema
+        self._dialect = db_connection._dialect
 
         # Link to Table Metadata.
         self._metadata_getter = TableMetadata(db_connection)
```

Comment:
Just a quibble, but - why not call this more directly after the class name, e.g.
```self._table_metadata = TableMetadata(db_connection)```
?
Lines 110 and 122 in dataset.py would seem more natural with a name like that.

Reply:
Name has been changed